README.md

```diff
@@ -14,17 +14,18 @@ Our Android template: **[template](https://github.com/nimblehq/android-templates
 ## Setup
 
 1. Clone or download this repository to your local machine, then extract and open the folder
-2. Install [Kscript](https://github.com/holgerbrandl/kscript#installation)
-3. Run `cd scripts` to get into the scripts directory
-4. Run `kscript new_project.kts` to create a new project with the following arguments:
+2. Install [Kotlin](https://github.com/JetBrains/kotlin) v1.6.21
+3. Install [Kscript](https://github.com/holgerbrandl/kscript#installation) v4.0.3
```

Comment:
I'd suggest moving to install Kotlin and Kscript to `Prerequisite` for executing preparation.